src/App.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
Why is this preferred over the normal `div`?

Reply:
@Duncanian I preferred `Fragment` because it doesn't add extra nodes to the DOM  but `div` does.